### PR TITLE
[DEV][ENHANCEMENT] Add trakt movie poster.

### PR DESCRIPTION
**Pull request by crawln45**
Added movie poster to trakt metadata.

### CODE REVIEW

File: flexget/plugins/api_trakt.py [hunk 1]
```diff
@@ -211,6 +212,8 @@ def get_api_url(*endpoint):
 def upgrade_database(ver, session):
     if ver <= 2:
         raise db_schema.UpgradeImpossible
+    if ver <= 3:
+        table_add_column('trakt_movies', 'poster', Unicode, session)
```
Comment (liiight) [inline flexget/plugins/api_trakt.py]: You didn't set the new version, `ver =4`. Check out series plugin for an example

File: flexget/plugins/api_trakt.py [hunk 2]
```diff
@@ -19,10 +19,11 @@
 from flexget.utils import requests
 from flexget.utils.tools import TimedDict
 from flexget.utils.database import with_session
+from flexget.utils.sqlalchemy_utils import table_add_column
 from flexget.utils.simple_persistence import SimplePersistence
 from flexget.logger import console
 
-Base = db_schema.versioned_base('api_trakt', 3)
+Base = db_schema.versioned_base('api_trakt', 4)
```
Comment (crawln45) [inline flexget/plugins/api_trakt.py]: @liiight Version 4

Comment (crawln45) [inline flexget/plugins/api_trakt.py]: oh and now i actually have some idea of how to do this ty

File: flexget/plugins/api_trakt.py [hunk 3]
```diff
@@ -423,6 +432,7 @@ def update(self, trakt_show, session):
         self.tmdb_id = trakt_show['ids']['tmdb']
         self.tvrage_id = trakt_show['ids']['tvrage']
         self.tvdb_id = trakt_show['ids']['tvdb']
+        self.poster = trakt_show['images']['poster']['full']
```
Comment (liiight) [inline flexget/plugins/api_trakt.py]: Can't this raise an exception if no poster is found?

**Pull request merged**